src/Components/SearchForm/SearchForm.jsx

```diff
@@ -1,15 +1,24 @@
 import { useState } from 'react';
 import styles from './SearchForm.module.css';
+import { useSearchParams } from 'react-router-dom';
+import { queryParamName } from '../../constants';
 
-export const SearchForm = ({ initialSearchQuery, onSearch }) => {
-  const [searchValue, setSearchValue] = useState(initialSearchQuery);
+export const SearchForm = () => {
+  const [searchParams, setSearchParams] = useSearchParams({ query: '' });
```

Comment:
You can use a constant for the param name here as well:

```suggestion
  const [searchParams, setSearchParams] = useSearchParams({ [queryParamName]: '' });
```

Reply:
And the same goes for other places where you are using `useSearchParams(...)`.